src/main/drivers/bus_spi.c

```diff
@@ -206,14 +206,14 @@ bool spiIsBusBusy(SPI_TypeDef *instance)
 
 }
 
-bool spiTransfer(SPI_TypeDef *instance, uint8_t *out, const uint8_t *in, int len)
```

Comment:
Rather than `out` and `in` can we use `rxData` and `txData` as in some of the other functions?